packages/manager/cypress/e2e/core/cloudpulse/alerts-listing-page.spec.ts

```diff
@@ -0,0 +1,290 @@
+/**
+ * @file Integration Tests for the CloudPulse Alerts Listing Page.
+ * This file contains integration tests to verify the functionality and UI elements
+ * on the CloudPulse  Alerts Listing Page.
+ */
+import { mockAppendFeatureFlags } from 'support/intercepts/feature-flags';
+import { accountFactory, alertFactory } from 'src/factories';
+import { mockGetAccount } from 'support/intercepts/account';
+import type { Flags } from 'src/featureFlags';
+import {
+  mockGetAllAlertDefinitions,
+  mockGetCloudPulseServices,
+} from 'support/intercepts/cloudpulse';
+import { formatDate } from 'src/utilities/formatDate';
+import { Alert } from '@linode/api-v4';
+import { ui } from 'support/ui';
+
+const flags: Partial<Flags> = { aclp: { enabled: true, beta: true } };
+const mockAccount = accountFactory.build();
+const now = new Date();
+const expectedHeaders = [
+  'Alert Name',
+  'Service',
+  'Status',
+  'Last Modified',
+  'Created By',
+];
+const alertDetails = [
+  alertFactory.build({
+    service_type: 'dbaas',
+    severity: 1,
+    status: 'enabled',
+    type: 'system',
+    created_by: 'user1',
+    updated: new Date(now.getTime() - 1 * 24 * 60 * 60 * 1000).toISOString(),
+  }),
+  alertFactory.build({
+    service_type: 'dbaas',
+    severity: 0,
+    status: 'disabled',
+    updated: new Date(now.getTime() - 10 * 24 * 60 * 60 * 1000).toISOString(),
+    created_by: 'user4',
+  }),
+  alertFactory.build({
+    service_type: 'linode',
+    severity: 2,
+    status: 'enabled',
+    updated: new Date(now.getTime() - 6 * 24 * 60 * 60 * 1000).toISOString(),
+    created_by: 'user2',
+  }),
+  alertFactory.build({
+    service_type: 'linode',
+    severity: 3,
+    status: 'disabled',
+    type: 'user',
+    updated: new Date(now.getTime() - 4 * 24 * 60 * 60 * 1000).toISOString(),
+    created_by: 'user3',
+  }),
+];
+/**
+ * Verifies the sorting functionality of a table column.
+ * This function:
+ * - Clicks the specified column header to trigger sorting.
+ * - Validates that the `aria-sort` attribute of the column header matches the given sort order.
+ * - Extracts cell values from the table and compares their order with the expected values.
+ *
+ * @param {string} columnDataQa - The `data-qa-header` attribute of the column to be sorted.
+ * @param {'ascending' | 'descending'} sortOrder - The expected sorting order ('ascending' or 'descending').
+ * @param {number[]} expectedValues - An array of expected values to validate against the sorted column.
+ */
+
+ function verifyTableSorting(
+  columnDataQa: string,
+  sortOrder: 'ascending' | 'descending',
+  expectedValues: number[]
+) {
```

Comment:
```suggestion
 const verifyTableSorting=(
  columnDataQa: string,
  sortOrder: 'ascending' | 'descending',
  expectedValues: number[]
) {
```

lets use this way of defining a function as it is used everywhere in the code